README.md

```diff
@@ -1 +1,2 @@
-# std-toast
\ No newline at end of file
+# std-toast
+This document scopes out a new HTML element for a 'toast' pop-up notification.
```

Comment:
Maybe stay more general at this stage, and go with "web platform API" instead of "HTML element".

Reply:
Got it
```suggestion
This document scopes out a web platform API for a 'toast' pop-up notification.
```